vision_agent/tools/tools.py

```diff
@@ -852,6 +852,38 @@ def ixc25_image_vqa(prompt: str, image: np.ndarray) -> str:
     return cast(str, data["answer"])
 
 
+def docqa_image(prompt: str, image: np.ndarray) -> str:
```

Comment:
I would change the input from `image: np.ndarray` to `images: list[np.ndarray]` to support multiple page documents in the future, even if the vision-agent does not support it yet.

Reply:
Done